interfaces.go

```diff
@@ -158,13 +158,15 @@ type Device interface {
 	IPAddresses() []string
 	Zone() Zone
 
-	// Parent, Owner, MAC Addresses if needed
+	// Parent returns the SystemID of the Parent. Most often this will be a
+	// Machine.
+	Parent() string
 
-	// InterfaceSet returns all the interfaces for the Device. This is an
-	// interim call until r4900 is packaged, when we will be able to remove the
-	// error response as the device JSON will include the interface set which
-	// will be able to be parsed at object creation time.
-	InterfaceSet() ([]Interface, error)
+	// Owner is the username of the user that created the device.
+	Owner() string
+
+	// InterfaceSet returns all the interfaces for the Device.
+	InterfaceSet() []Interface
```

Comment:
Because it was called "interface_set" in the JSON response. That is the only reason I chose that name.